op-service/tls/cli.go

```diff
@@ -106,5 +109,6 @@ func ReadCLIConfigWithPrefix(ctx *cli.Context, flagPrefix string) CLIConfig {
 		TLSCaCert: ctx.String(prefixFunc(TLSCaCertFlagName)),
 		TLSCert:   ctx.String(prefixFunc(TLSCertFlagName)),
 		TLSKey:    ctx.String(prefixFunc(TLSKeyFlagName)),
+		Enabled:   ctx.IsSet(TLSCaCertFlagName) || ctx.IsSet(TLSCertFlagName) || ctx.IsSet(TLSKeyFlagName),
```

Comment:
This previously was an and check for each

Reply:
It may have been intended as an `and` check & I'm happy to swap it over, but in practice the code always uses TLS unless every value is explicitly set to the empty string.

Reply:
No need for extra back and forth, ty @trianglesphere